cve_bin_tool/checkers/ffmpeg.py

```diff
@@ -26,7 +26,7 @@ def get_version(lines, filename):
     contains_ffmpeg = "Codec '%s' is not recognized by FFmpeg." in lines
     version_regex = [r"%s version ([0-9]+\.[0-9]+\.[0-9]+)"]
     version_info = dict()
-    if filename[::-1].startswith(("ffmpeg")[::-1]):
+    if "ffmpeg" in filename.lower():
```

Comment:
Should this still be startswith?  I'm asking because I know ffmpeg has some plugin system, and on debian at least they seem to get names like `cmus-plugin-ffmpeg` and i can't tell at a glance whether those should be detected as ffmpeg or not.